pkg/gatewayserver/gatewayserver.go

```diff
@@ -44,6 +44,9 @@ import (
 	"google.golang.org/grpc"
 )
 
+// connConcurrentUplinks is the number of goroutines per gateway connection to handle upstream messages.
+var connConcurrentUplinks = 16
```

Comment:
Nitpick: `concurrentUplinksPerConnection` feels more readable.

Reply:
We're addressing a better solution after release, still this month; #464